src/lib/storage/chunk.hpp

```diff
@@ -161,13 +161,13 @@ class Chunk : private Noncopyable {
    * However, `size() - invalid_row_count()` does not necessarily tell you how many rows are visible for
    * the current transaction.
    */
-  uint32_t invalid_row_count() const { return _invalid_row_count.load(); }
+  ChunkOffset invalid_row_count() const { return _invalid_row_count.load(); }
```

Comment:
This is not really what we want here but, as discussed yesterday, creating a strong typedef for ChunkOffset will take much more time and this might be a bit less wrong here.